apps/editor/src/app/editor-app.component.html

```diff
@@ -1,3 +1,4 @@
+<gd-loading-mask [loadingMask]="isLoading"></gd-loading-mask>
```

Comment:
Maybe we should add this to the viewer application too?

Reply:
Good question! Also i remembered that we probably could remove now inline loading masks like [this](https://github.com/groupdocs-total/GroupDocs.Total-Angular/blob/3b3a3994b80afd793d2a3475039959d9ed6c4027/libs/common-components/src/lib/page/page.component.html#L7) and [this](https://github.com/groupdocs-total/GroupDocs.Total-Angular/blob/e37e2a45da8f4d6560a562004164499eac25d1ef/libs/common-components/src/lib/browse-files-modal/browse-files-modal.component.html#L43)? 
What do you think?

cc @egorovpavel

Reply:
The first one - we need it for page loading after we got data from the server and before the rendering has finished, about the second - i dont remember why we need it, maybe we can remove it

Reply:
We can remove the [second](https://github.com/groupdocs-total/GroupDocs.Total-Angular/blob/e37e2a45da8f4d6560a562004164499eac25d1ef/libs/common-components/src/lib/browse-files-modal/browse-files-modal.component.html#L43) one.